checkstyle.xml

```diff
@@ -109,6 +109,9 @@
 		<module name="RedundantImport">
 			<property name="severity" value="error"/>
 		</module>
+		<module name="UnusedImports">
+			<property name="severity" value="error"/>
+		</module>
```

Comment:
This is not my commit. Before squashing my commits, i ran the below command 
> git pull --rebase upstream master
 to get the latest code and this pulled checkstyle.xml change, I think this change is committed by @dkayiwa